portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,263 @@
 }
 
-#greeting-container {
-  margin-top: 20px;
+body {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
 }
+
+main section.home {
+    padding-top: 60px;
+    margin-top: 0px;
+}
+
+main section {
+    margin: 10px 0 10px 0;
+}
+
+a {
+    color: #22C9E0;
+    text-decoration: none;
+}
+
+a:hover, a:focus, a:active {
+    text-decoration: underline;
+}
+
+/* navigation bar */ 
+
+nav {
+    background-color: #22C9E0;
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    align-items: center;
+}
+
+.nav__logo {
+    padding-left: 50px;
+    padding-right: 50px;
+}
+
+.nav__items {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    list-style-type: none;
+    margin: 0;
+    padding: 0 100px 0 0;
+    width: 100%;
+}
+
+.nav__item {
+    padding: 0 25px 0 25px;
+    font-size: 17pt;
+    color: white;
+    border-bottom: 1px solid transparent;
+    padding-bottom: 1px;
+}
+
+.nav__item:hover, .nav__item:focus {
+    border-bottom: 2px solid white;
+    padding-bottom: 1px;
+}
+
+/* home section */
+
+section.home {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+section.home img {
+    width: 300px;
+    height: auto;
+    padding: 30px;
+    border-radius: 20%;
+}
+
+section.home p {
+    font-size: 28pt;
+    padding: 30px;
+}
+
+/* projects section */
+
+section.projects {
+    background: #F4F6F7;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+}
+
+.projects__names {
+    list-style-type: none;
+    padding-right: 50px;
+    padding-left: 30px;
+}
+
+.projects__names li {
+    padding: 5px 0 5px 0;
+    width: 300px;
+    text-align: center;
+    font-size: 20px;
+}
+
+.project.active, .project:hover, .project:focus {
+    background: #22C9E0;
+    color: white;
+}
+
+.projects__descriptions {
+    padding-right: 30px;
+}
+
+.project_description {
+    display: none;
+    padding-left: 25px;
+}
+
+.project_description.active {
+    display: block;
+    font-size: 16pt;
+}
+
+/* skills section */
+
+section.skills {
+    padding-left: 30px;
+}
+
+section.skills h3 {
+    font-size: 26px;
+}
+
+section.skills p {
+    font-size: 18px;
+    padding-left: 10px;
+}
+
+/* cats section */
+
+section.cats {
+    background: #F4F6F7;
+}
+
+section.cats ul {
+    list-style-type: none;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-auto-rows: 1fr;
+    padding: 0;
+    margin: 0;
+}
+
+.cat {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+}
+
+.cat:hover img {
+    opacity: 0;
+}
+
+.cat:hover .cat_description {
+    opacity: 1;
+}
+
+.cat img, .cat_description {
+    height: auto;
+    display: block;
+    margin: 0 auto;
+    transition: opacity 0.5s ease-in-out;
+}
+
+.cat img {
+    width: 300px;
+    display: block;
+    margin: 0 auto;
+}
+
+.cat_description {
```

Comment:
From the [style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters), it's recommended to use hyphen (`-`) instead of underscore (`_`) to separate ID and class names. So in this case, `cat-description` would be a better naming. The same applies to other elements.